src/plugin/ethAction.ts

```diff
@@ -153,9 +145,9 @@ export class EthereumActionHelper {
     this.updateActionProperty('gasLimit', valueHex)
   }
 
-  /** set gasPrice - value should be a decimal string in units of GWEI e.g. '123' */
+  /** set gasPrice - value should be a string in units of WEI or a Hex value e.g. '123' */
```

Comment:
This is all confusing so please provide a helpful example in comments
The comment that was here before used 'decimal' to imply a string like '0.0001'. 
Please update the examples in comment so it provides one of each (string with decimals '0.0001') or hexstring '...')

Reply:
done.